pkg/controller/controller.go

```diff
@@ -75,7 +76,17 @@ func (c *Controller) Start() error {
 	if c.client.workloadController != nil {
 		c.client.workloadController.Processor.Sm = secertManager
 	}
+	clientset, err := utils.GetK8sclient()
```

Comment:
Can the clientset be reused with bypass? and does the kmesh manager logic need to be moved before the bypass logic.